Packs/ReversingLabs_A1000/Integrations/ReversingLabsA1000v2/README.md

```diff
@@ -3396,3 +3074,895 @@ Search for hashes on A1000 using multi-part search criteria
+                        "update_time": "2023-05-18T01:32:36"
+                    }
+                ],
+                "statistics": {
+                    "clean": 0,
+                    "malicious": 1,
+                    "total": 13,
+                    "undetected": 12
+                }
+            },
+            "top_threats": [
+                {
+                    "files_count": 1,
+                    "risk_score": 10,
+                    "threat_name": "Win64.Trojan.Casdet"
+                },
+                {
+                    "files_count": 1,
+                    "risk_score": 10,
+                    "threat_name": "ByteCode-MSIL.Backdoor.DCRat"
+                },
+                {
+                    "files_count": 1,
+                    "risk_score": 10,
+                    "threat_name": "ByteCode-MSIL.Infostealer.RedLine"
+                },
+                {
+                    "files_count": 1,
+                    "risk_score": 10,
+                    "threat_name": "Win32.Trojan.Fragtor"
+                }
+            ]
+        }
+    }
+}
+```
+
+#### Human Readable Output
+
+>## ReversingLabs A1000 Domain Report for ink-scape.online
+> **Modified time**: 2023-05-18T06:06:19
+> ### Top threats
+>|files_count|risk_score|threat_name|
+>|---|---|---|
+>| 1 | 10 | Win64.Trojan.Casdet |
+>| 1 | 10 | ByteCode-MSIL.Backdoor.DCRat |
+>| 1 | 10 | ByteCode-MSIL.Infostealer.RedLine |
+>| 1 | 10 | Win32.Trojan.Fragtor |
+>
+> ### Downloaded files statistics
+> **Unknown**: 0
+>    **Suspicious**: 0
+>    **Malicious**: 4
+>    **Goodware**: 0
+>    **Total**: 4
+>    
+>**Last DNS records time**: 2023-05-11T17:46:01
+>    
+> ### Last DNS records
+>|provider|type|value|
+>|---|---|---|
+>| ReversingLabs | A | 37.140.192.210 |
+>
+> ## Third party reputations
+> ### Statistics
+> **Malicious**: 1
+>    **Undetected**: 12
+>    **Clean**: 0
+>    **Total**: 13
+>    
+> ### Sources
+>|detection|source|update_time|
+>|---|---|---|
+>| undetected | phishing_database | 2023-05-18T01:26:00 |
+>| undetected | 0xSI_f33d | 2023-05-18T05:21:25 |
+>| undetected | cyradar | 2023-05-17T06:16:21 |
+>| undetected | adminus_labs | 2023-05-18T05:40:37 |
+>| undetected | apwg | 2023-05-18T05:35:59 |
+>| malicious | netstar | 2023-05-18T06:06:19 |
+>| undetected | threatfox_abuse_ch | 2023-05-17T23:20:08 |
+>| undetected | botvrij | 2023-05-18T01:25:37 |
+>| undetected | alphamountain | 2023-05-18T06:00:06 |
+>| undetected | comodo_valkyrie | 2023-05-18T04:52:55 |
+>| undetected | web_security_guard | 2022-01-21T06:56:15 |
+>| undetected | osint | 2023-05-18T00:30:47 |
+>| undetected | crdf | 2023-05-18T01:32:36 |
+
+
+### reversinglabs-a1000-ip-address-report
+
+***
+Get a report for the submitted IP address.
+
+#### Base Command
+
+`reversinglabs-a1000-ip-address-report`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| ipAddress | IP address string. | Required | 
```

Comment:
So the argument names need to be snake case instead of camel case?

Reply:
For the first comment, yes, the arguments of the new commands should be in snake case, instead of camel case.
As. for the second comment, yes, some of the new commands have too many results, which makes the README file very large. There is no need for so many results, so please remove the unnecessary and leave only 2-3 results for each command.